features/support/topic_helper.rb

```diff
@@ -52,9 +38,17 @@ def stub_topic_lookups
         ],
       },
       links: {
-        "parent" => [
-          "title" => "Oil and Gas",
-          "base_path" => "/oil-and-gas",
+        parent: [
+          { title: "Oil and Gas", base_path: "/oil-and-gas" }
+        ],
+        topic_content: [
```

Comment:
Now that we're only using content store data, we can use the `govuk_schemas` gem to validate fixtures like this, so they won't get out of date. Could you see if https://alphagov.github.io/govuk_schemas_gem/GovukSchemas/RandomExample.html#merge_and_validate-instance_method would work?